pingpong.c

```diff
@@ -197,6 +198,13 @@ int main(int argc, char **argv)
         exit(1);
       }
       break;
+    case 'c':
+      nr_tested_cores = strtoul(optarg, &p, 0);
+      if (*p) {
+        fprintf(stderr, "-c requires a numeric argument\n");
+        exit(1);
+      }
+      break;
```

Comment:
Please add to the default: printout case as well, so the user can tell that there is this new option:
fprintf(stderr, "usage: %s [-l | -u | -x] [-r nr_relax] [-s nr_array_elts_to_dirty]\n", argv[0]);

Reply:
Done. Thanks to you too, Stephane.